### PR TITLE
Update `GB Cup 2024 Spring` wiki articles

### PR DESCRIPTION
**Pull request by HanHan3z**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Tournaments/GBC/GBC_2024_Spring/en.md [hunk 1]
```diff
@@ -29,18 +29,36 @@ The **GB Cup 2024 Spring** (***GBC 2024 Spring***) is an osu!mania 4K point-base
...
-| *13th place – 16th place* | *TBA* RMB |
-| *17th place – 24th place* | *TBA* RMB |
-| *Full attendance prize* | KanonBot profile badge (all groups), *TBA* RMB (Group B), *TBA* RMB (Group A), *TBA* RMB (Group S), *TBA* RMB (Group EX) |
+| ![Gold crown](/wiki/shared/crown-gold.png "playoff stage 1st place") | Profile badge, 6 months supporter |
+| ![Silver crown](/wiki/shared/crown-silver.png "playoff stage 2nd place") | 6 months supporter |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "playoff stage 3rd place") | 4 months supporter |
+| *4th place – 6th place* | $10 |
+| *7th place – 12th place* | 2 months supporter |
+| *13th place – 24th place* | $6 |
+| *Full attendance prize* | $2 (Group A), $3 (Group S), 1 month supporter (Group EX) |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2024_Spring/en.md:59]:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "playoff stage 1st place") | Profile badge, 6 months of osu!supporter |
| ![Silver crown](/wiki/shared/crown-silver.png "playoff stage 2nd place") | 6 months of osu!supporter |
| ![Bronze crown](/wiki/shared/crown-bronze.png "playoff stage 3rd place") | 4 months of osu!supporter |
| *4th place – 6th place* | $10 |
| *7th place – 12th place* | 2 months of osu!supporter |
| *13th place – 24th place* | $6 |
| *Full attendance prize* | $2 (Group A), $3 (Group S), 1 month supporter (Group EX) |
```

Comment (Walavouchey) [reply wiki/Tournaments/GBC/GBC_2024_Spring/en.md:59]: ... is this correct? cash prizes between 4th-24th + attendance, but not for 7th-12th and 2nd-3rd?

seems a bit random so i want to confirm

Comment (HanHan3z) [reply wiki/Tournaments/GBC/GBC_2024_Spring/en.md:59]: 2 months of osu!supporter is equivalent to 8 USD and 1 month of osu!supporter is equivalent to 4 USD
im not sure if this is clear enough, asking the editor now

Comment (HanHan3z) [reply wiki/Tournaments/GBC/GBC_2024_Spring/en.md:59]: adjusted the prize description to a specific amount

Comment (yobrevelc) [reply wiki/Tournaments/GBC/GBC_2024_Spring/en.md]: > ... is this correct? cash prizes between 4th-24th + attendance, except 7th-12th and not for 2nd-3rd?
> 
> seems a bit random so i want to confirm

only full-attendance player will be able to participate in playoff, so the prize of playoff already include the full-attendance prize

Comment (yobrevelc) [reply wiki/Tournaments/GBC/GBC_2024_Spring/en.md]: > ... is this correct? cash prizes between 4th-24th + attendance, except 7th-12th and not for 2nd-3rd?
> 
> seems a bit random so i want to confirm

only full-attendance player will be able to participate in playoff, so the prize of playoff already include the full-attendance prize

**Comment by HanHan3z** [comment]
Adding mappool details

**Comment by HanHan3z** [comment]
all work completed
waiting for review now

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**